ops/plotting.py

```diff
@@ -326,6 +387,9 @@ def two_feature(
             adjust_text(labels, df_[x].values, df_[y].values, ax=ax, force_text=(0.1, 0.05), force_points=(0.01, 0.025))
         except:
             pass
+    
+    if save_plot_path:
+        plt.savefig(save_plot_path, dpi=300, bbox_inches='tight')
```

Comment:
Thanks for adding this. I think that when we move the eval code to the smk process we will need every plotting function to have this since the user will no longer be able to view the plots in a notebook.

Reply:
Yeah. We need to rewrite the qc code to output a file. Shouldn't be that hard. Or just to add a wrapper that saves figs in the snakemake code. But for now, we aren't really using those functions in notebooks.